### PR TITLE
Shell script integration

### PR DESCRIPTION
**Pull request by jc-rh**

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ configMapGenerator:
   name: manager-config
 images:
 - name: controller
-  newName: quay.io/openshift-kni/lifecycle-agent-operator
+  newName: quay.io/junchen/lifecycle-agent-operator
```
Comment (donpenney) [inline config/manager/kustomization.yaml:15]: Should drop this change

File: controllers/prep_handlers.go [hunk 2]
```diff
@@ -18,28 +18,97 @@ package controllers
...
+		var content []byte
+		content, err = os.ReadFile(progressfile)
+		if err != nil {
+			return
+		}
+
+		progress := strings.TrimSpace(string(content))
+		r.Log.Info("Prep progress: " + progress)
+
+		if progress == "Completed" {
```
Comment (donpenney) [inline controllers/prep_handlers.go:74]: We can add a check for "Failed" as well

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (jc-rh) [reply config/manager/kustomization.yaml]: Done

File: controllers/prep_handlers.go (hunk 2 above)
Comment (jc-rh) [reply controllers/prep_handlers.go:74]: Added

**Pull request merged**